Assets/MixedRealityToolkit/_Core/Interfaces/Devices/IMixedRealityControllerVisualizer.cs

```diff
@@ -15,6 +15,11 @@ public interface IMixedRealityControllerVisualizer : IMixedRealityControllerPose
         /// </remarks>
         GameObject GameObjectProxy { get; }
 
+        /// <summary>
+        /// The current controller reference.
+        /// </summary>
+        IMixedRealityController Controller { get; set; }
```

Comment:
I see no reason to attach a physical controller in to the abstract implementation for visualization.

The events already give access to the source controller / data.

And the current visualizer has dependencies on the SourcePoseSynchroniser

Reply:
This is needed for a line in `BaseController`, which only references this interface: https://github.com/Microsoft/MixedRealityToolkit-Unity/blob/mrtk_development/Assets/MixedRealityToolkit/_Core/Devices/BaseController.cs#L197